constants/chainIds.json

```diff
@@ -1,19 +1,71 @@
 {
-  "ethereum":         101,
-  "bsc":              102,
-  "avalanche":        106,
-  "polygon":          109,
-  "arbitrum":         110,
-  "optimism":         111,
-  "fantom":           112,
+  "ethereum": 101,
+  "bsc": 102,
+  "avalanche": 106,
+  "aptos": 108,
+  "layerzero_apps": 108,
+  "polygon": 109,
+  "arbitrum": 110,
+  "optimism": 111,
+  "fantom": 112,
+  "dfk": 115,
+  "harmony": 116,
+  "dexalot": 118,
+  "celo": 125,
+  "moonbeam": 126,
+  "fuse": 138,
+  "gnosis": 145,
+  "klaytn": 150,
+  "metis": 151,
+  "coredao": 153,
+  "okt": 155,
+  "polygon-zkevm": 158,
+  "canto": 159,
+  "zksync-era": 165,
+  "moonriver": 167,
+  "tenet": 173,
+  "arbitrum-nova": 175,
+  "meter.io": 176,
+  "sepolia": 161,
+  "kava": 177,
+  "linea": 183,
+  "base": 184,
+  "mantle": 181,
 
-  "goerli":           10121,
-  "bsc-testnet":      10102,
-  "fuji":             10106,
-  "mumbai":           10109,
-  "arbitrum-goerli":  10143,
-  "optimism-goerli":  10132,
-  "fantom-testnet":   10112,
-  "meter-testnet":    10156,
-  "zksync-testnet":   10165
-}
\ No newline at end of file
+  
+  "goerli": 10121,
+  "bsc-testnet": 10102,
+  "fuji-avalanche": 10106,
+  "aptos": 10108,
```

Comment:
some of the keys are duplicated.
it seems it should have the suffix something like `-testnet` for testnets.
e.g. `"aptos-testnet": 10108`